### ISSUE TITLE
How to make noninteractive plots from explain_global?

### ISSUE THREAD

**Issue by lambdaofgod** [interpretml/interpret#297]
I run `interpret` in GCP Jupyter and 

```

ebm_global = ebm.explain_global()
show(ebm_global)

```

Doesn't show anything.
The simplest method to sidestep this would be to call some method that just takes id or feature name and plots what's outputted from selector when selecting it.

Is there a way of doing this?

**Comment by apatange-source** [comment]
Check issue #290 
Is this what you're talking about?

**Comment by lambdaofgod** [comment]
Not really. As far as I know plotly figures aren't visible in jupyter notebooks.

**Comment by paulbkoch** [comment]
I believe you would need to use the inline provider there, obtained with:

```
from interpret import set_visualize_provider
from interpret.provider import InlineProvider
set_visualize_provider(InlineProvider())
```

More on this in the docs: https://interpret.ml/docs/deployment-guide.html
